app/models/jam.rb

```diff
@@ -2,6 +2,7 @@
 
 class Jam < ApplicationRecord
   belongs_to :room
+  belongs_to :user, optional: true
```

Comment:
I assume it's optional because we expect to keep jams even if their creator was deleted not because we expect to create jams by non-users?